### PR TITLE
Added `Multiple days selection` feature & `dayBtnProps` prop

### PR DESCRIPTION
**Pull request by smrsan76**
This feature is what you've recently wrote for me to complete one of my tasks. (with a little bug fixes and improvements) :wink: 

Screenshot from 2019-07-11 03-58-06

### CODE REVIEW

**Comment by smrsan76** [comment]
Oh ... I've just added another commit to this PR. Please merge it, too. Because it's one of our important needs for our project to manipulate day buttons.

E.g. we wanna add a `hover` functionality for each day button, and/or e.g. we wanna find the x,y coordinates of each day button elements when they are being hovered.

Generally, there is no better solution to implement such a specific business needs unless we specify a way to manipulate some calendar elements directly. (specially for day buttons!)

**Comment by smrsan76** [comment]
Dear @Kiarash-Z 
I have some questions about "How to request multiple changes at once?"
Am I able to commit multiple changes through this PR?
Because all of them are generally related to the title of this PR.

If I'm not able to commit multiple changes through here, what is your suggestion for requesting multiple changes?

Can I create PRs from branches other than `develop`? (e.g. `branch name = feature name`)

**Comment by thebrodmann** [comment]
Hey @smrsan76, I'm not the maintainer but as some of your questions refer to the best practices and generally has a common answer, I try to help you.
Maybe the maintainers of the project, not be agreed with my answers. So before any decision, please wait for their answers.

> How to request multiple changes at once?

**TL;DR** try to break the changes as you can into PRs unless the maintainer request from you to do something else.

It depends on changes. If they are tightly coupled together and they live upon **all or none principle**, you should request them in one PR. However, if the changes can have meaning without together it's better to break them into multiple PRs.

> Can I create PRs from branches other than develop?

Usually, you wanna apply changes on the most stable version of the repo, so you should fork from the master branch and PR to it, then maintainers change the base branch to whatever they want.

**Comment by smrsan76** [comment]
Hey @thebrodmann 
Thanks for writing this amount of guides for us. (me + other developers)

> Usually, you wanna apply changes on the most stable version of the repo, so you should fork from the master branch and PR to it, then maintainers change the base branch to whatever they want.

Unfortunately/Fortunately I have so many proposals and changes to commit via multiple pull requests, and "multiple pull requests" means that I need more than 2 branches. (other than `develop` & `master`)
Specially when @Kiarash-Z is too busy and I wouldn't request more than 2 and I have to wait for @Kiarash-Z to come and merge them.

So, I wanna find a way to commit more than 2 changes using one of these approaches:
- Committing through PRs of multiple custom branches.
- Committing through a single PR. (I don't recommend it).
- A new approach which you would recommend/introduce to me.

**Comment by thebrodmann** [comment]
> I have so many proposals

I suggest open issues for all of them before implementing. In this way, not only you don't waste your time but also the community and maintainers can vote or comment on your proposals or just suggest better ways to implement. Also, it's possible to find other people yearn for helping you in implementation.

> multiple pull requests means that I need more than 2 branches

I think you are overthinking. Just fork the repo from the master branch; then fork how many branch you want from the master branch of your repo and apply your changes. When completed just PR them to the master branch of this repo.

**Comment by Kiarash-Z** [comment]
@thebrodmann thanks for responding in my absence. highly appreciated...❤️

**Comment by Kiarash-Z** [comment]
@smrsan76 Due to very rude behavior in Telegram, You got blocked...
sd
